examples/stabilizer_code.py

```diff
@@ -0,0 +1,340 @@
+from typing import Dict, List, Tuple
+
+import numpy as np
+
+import cirq
+
+# Based on:
+# Stabilizer Codes and Quantum Error Correction
+# Daniel Gottesman
+# https://thesis.library.caltech.edu/2900/2/THESIS.pdf
+#
+# However, there were some mistakes in the thesis:
+# https://www2.perimeterinstitute.ca/personal/dgottesman/thesis-errata.html
+#
+# An alternative explanation can be found in exercises 10.3 and 10.4 of
+# "Quantum Computation and Quantum Information" by Michael Nielsen and Isaac Chuang.
+
+
+def _BuildByCode(mat: np.ndarray) -> List[str]:
```

Comment:
Curious: Why is the name of this function like this instead of `build_by_code`?

Reply:
Good point, I wasn't consistent with my naming convention. I think, though, that it should have an underscore first, but I otherwise renamed it. I took the liberty of also renaming GaussianElimination since it was suffering from the same inconsistency. Hope that it's OK.